pkg/google/pubsub.go

```diff
@@ -69,13 +62,13 @@ type logMessage struct {
 	} `json:"protoPayload"`
 }
```

Comment:
Apparently the project ID exists as a label in the message payload from the pubsub message. Could be fetched by unmarshalling to the following struct:

```go
type logMessage struct {
	Timestamp    time.Time `json:"timestamp"`
	ProtoPayload struct {
		ResourceName       string `json:"resourceName"`
		AuthenticationInfo struct {
			PrincipalEmail string `json:"principalEmail"`
		} `json:"authenticationInfo"`
	} `json:"protoPayload"`
	Resource struct {
		Labels struct {
			ProjectID string `json:"project_id"`
		} `json:"labels"`
	} `json:"resource"`
}
```